doc-content/enterprise-only/processes/filtering-process-instances-proc.adoc

```diff
@@ -0,0 +1,49 @@
+[id='filtering-process-instances-proc']
+= Filtering process instances using basic or advanced filters
+
+{Central} now provides you with basic and advanced filters to help you filter and search through running process instances. You can filter processes by attributes such as *State*, *Errors*, *Filter By*, *Name*, *Start Date*, and *Last update*. You can also create custom filters using the *Advanced Filters* option. The newly created custom filter is added to the *Saved Filters* pane, which is accessible by clicking on the star icon on the left of the *Manage Process Instances* page.
+
+[float]
+== Filtering process instances using basic filters
+
+.Procedure
+. Go to *Menu* -> *Manage* -> *Process Instances*.
+. On the *Manage Process Instances* page, click the filter icon on the left of the page to expand the *Filters* pane.
++
+This pane lists the following process attributes which you can use to filter process instances:
++
+* *State*: Filter process instances based on their state (*Active*, *Aborted*, *Completed*, *Pending*, and *Suspended*).
+* *Errors*: Filter process instances that contain at least one or no errors.
+* *Filter By*: Filter process instances based on *Id*, *Initiator*, *Correlation Key*, or *Description* attribute.
++
+... Select the required attribute.
+... Enter the search query in the text field below.
+... Click *Apply*.
++
+
+* *Name*: Filter process instances based on process definition name.
+* *Start Date*: Filter process instances based on their creation date.
+* *Last update*: Filter process instances based on their last modified date.
```

Comment:
`... last modification date.` 
maybe?

Reply:
Left as is. Usage of either, last modified date or last modification date would do here. Here last modified date seems more appropriate.